demisto_sdk/commands/common/content/objects/pack_objects/pack.py

```diff
@@ -328,16 +328,26 @@ def is_server_version_ge(self, client, server_version_to_check):
         server_version = get_demisto_version(client)
         return LooseVersion(server_version.base_version) >= LooseVersion(server_version_to_check)  # type: ignore
 
-    def upload(self, logger: logging.Logger, client: demisto_client):
+    def upload(self, logger: logging.Logger, client: demisto_client, skip_validation: bool):
         """
         Upload the pack zip to demisto_client,
         from 6.5 server version we have the option to use skip_verify arg instead of server configuration.
         Args:
             logger (logging.Logger): System logger already initialized.
             client: The demisto_client object of the desired XSOAR machine to upload to.
+            skip_validation: if true will skip upload packs validation.
         Returns:
             The result of the upload command from demisto_client
         """
+        if self.is_server_version_ge(client, '6.6.0') and skip_validation:
+            try:
```

Comment:
It seems like for 6.6.0 two uploads will be made, for `is_server_version_ge(client, '6.5.0')` and for 6.6.0.
Is that intentional, or should we change the`if` on line 351 to `elif`?

Reply:
I can replace it to `elif`, but we have the return in line 345